resources/ory/charts/hydra/templates/hpa.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if .Values.deployment.autoscaling.enabled }}
+apiVersion: autoscaling/v2beta2
+kind: HorizontalPodAutoscaler
+metadata:
```

Comment:
We have a HPA for both oathkeeper and hydra in `resources/ory/templates/hpa.yaml`. We should think if we would like to remove it.